requirements/test.txt

```diff
@@ -506,6 +525,10 @@ networkx==3.2.1
     # via
     #   scikit-image
     #   torch
+nixl==0.8.0
+    # via lmcache
```

Comment:
this version is going to be conflicting with the one we have in requirements/kv_connectors.txt
Any chance we can avoid installing nixl through lmcache?